src/core/thread/topology.hpp

```diff
@@ -351,6 +351,14 @@ class Neighbor : public InstanceLocatorInit
      */
     void SetRloc16(uint16_t aRloc16) { mRloc16 = aRloc16; }
 
+    /**
+     * This method indicates whether or not it is a valid Thread 1.1 neighbor.
+     *
+     * @returns TRUE if it is a valid Thread 1.1 neighbor, FALSE otherwise.
+     *
+     */
+    bool IsThreadVersion2(void) const { return mState != kStateInvalid && mVersion == OT_THREAD_VERSION_1_1; }
```

Comment:
```suggestion
    bool IsThreadVersion1_1(void) const { return mState != kStateInvalid && mVersion == OT_THREAD_VERSION_1_1; }
```
Prefer `IsThreadVersion1_1` because we commonly say version 1.1, 1.2, we never say version 2, 3 in our conversations. I think `IsThreadVersion1_1` is more understandable to code readers.

Reply:
no strong preference. renamed as your suggestion.